lib/src/main/java/io/fabianterhorst/isometric/Isometric.java

```diff
@@ -150,6 +112,23 @@ public void measure(int width, int height, boolean sort, boolean cull, boolean b
         this.originX = width / 2;
         this.originY = height * 0.9;
 
+        transformItems(this.items, cull, boundsCheck);
+
+        if (sort) {
+            this.items = sortPaths();
+        }
+    }
+
+    //allow user to recalculate only certain items
+    public void updateItems(List<Item> items, boolean cull, boolean boundsCheck) {
```

Comment:
Aren't the interception checks for the none updated items missing when you update an item that is behind an none updated one?

Reply:
The user would be using this method to directly manipulate the items list returned from `getCurrentItems()`. I will add a comment that they are taking potentially dangerous actions when they use this method (manipulating an item that is covered by another item), but it is still a useful method to have. 

In my own implementation, I get the current items `List lis = getCurrentItems()` and find references to some `Shape s` I care about. I then create new `Item` into `List modItems` and use `updateItems(modItems, true, true)` to perform all of the path calculations with culling and bounds detection which removes the bad faces from the provided list. I then directly remove/replace the items `updateItemsList(lis, modItems)`